File: April 20, 2026.js
function findOrg(acronym) {

  const list = ["National Avocado Storage Authority", "Cats Infiltration Agency", "Fluffy Beanbag Inspectors", "Department Of Jelly", "Wild Honey Organization", "Eating Pancakes Administration"];

  for (let name of list) {
    
    let firstLetters = name.split(" ").map(name => name[0]).join("");
    if (firstLetters == acronym) {
      return name
    }
  }

  return "Not Found";
}

File: April 20, 2026.py
def find_org(acronym):

    names = ["National Avocado Storage Authority", "Cats Infiltration Agency", "Fluffy Beanbag Inspectors", "Department Of Jelly", "Wild Honey Organization", "Eating Pancakes Administration"];

    for name in names:
        first_letters = "".join(word[0] for word in name.split())
        print(first_letters)
        if first_letters == acronym:
            return name

    return "Not Found"